start-mongodb.sh

```diff
@@ -1,3 +1,18 @@
 #!/bin/sh
 
-docker run --name mongodb --publish 27017:27017 --detach mongo:$1
+REPL_COMMAND=""
+
+if [ -n "$MONGO_REPL_SET" ]
+then
+    REPL_COMMAND="--replSet $MONGO_REPL_SET"
+fi
+
+docker run --name mongodb --publish 27017:27017 --detach mongo:$1 $REPL_COMMAND
+
+if [ -n "$MONGO_REPL_SET" ]
+then
+    sleep 1
```

Comment:
Can you please explain why you added this `sleep 1` here?

Reply:
I think it's from an old recipe to make sure the server is started before initiating the replica set.

Let's take it out and see if it still works?